src/services/ZendeskService.php

```diff
@@ -72,7 +72,10 @@ public function submitTicket($data)
 		$output = self::curlWrap("/tickets.json", $create, $headers);
 		
 		//get the ticket ID - also checks the new ticket was created successfully
-		$ticketId = $output->ticket->id;
+                if($output->error) {
```

Comment:
Maybe you could also check the existence of `error` as class property. Otherwise an error is still going to be thrown.

Reply:
Like this? (see update)

Reply:
That's a way to do it 👍